stac_fastapi/elasticsearch/stac_fastapi/elasticsearch/app.py

```diff
@@ -60,6 +62,8 @@
 post_request_model = create_post_request_model(extensions)
 
 api = StacApi(
+    title=os.getenv("STAC_FASTAPI_TITLE", "stac-fastapi"),
+    description=os.getenv("STAC_FASTAPI_DESCRIPTION", "stac-fastapi"),
```

Comment:
Should we include the STAC_FASTAPI_VERSION from the pr in ` stac-fastapi? api_version: str = attr.ib(default=os.getenv("STAC_FASTAPI_VERSION", "0.1"))` Also, should these have more specific default values? The title here for instance could be stac-fastapi.elasticsearch. The api version would be 2.1. If we could pull the default version in from the version file?

Reply:
Yeah, good call. I will make those changes

Reply:
We could add the new env vars to the app-opensearch and app-elasticsearch in the docker-compose.yml file too. It might help people realize that they can change those things.

Reply:
Cool. Added that as well